lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2808,6 +2808,14 @@ void generateIntelFPGAAnnotation(const SPIRVEntry *E,
       Out << ":" << Str;
     Out << '}';
   }
+  if (E->hasDecorate(DecorationBankBitsINTEL)) {
+    Out << "{bank_bits:";
+    auto Literals = E->getDecorationLiterals(DecorationBankBitsINTEL);
+    for (size_t I = 0; I < Literals.size() - 1; ++I) {
+      Out << Literals[I] << ",";
+    }
+    Out << Literals[Literals.size() - 1] << '}';
```

Comment:
Would it make sense to factor this routine out into a separate function and re-use in `generateIntelFPGAAnnotationForStructMember()`? (Just asking)

Reply:
Maybe not, it is a story about just 3 common lines of code 🤔

Reply:
Ah, of course. My bad.